collab-document/src/blocks/entities.rs

```diff
@@ -50,11 +56,14 @@ pub struct BlockAction {
 #[derive(Debug, Clone, Serialize)]
 pub struct BlockActionPayload {
   /// Block
-  pub block: Block,
+  pub block: Option<Block>,
```

Comment:
Please add a comment here to describe why it can be optional. 

As far as I know, it can be null when we only need to update the delta/text-id.